lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -272,14 +290,17 @@ class CategoryRenderer {
    */
   renderUnexpandableClump(auditRefs, groupDefinitions) {
     const clumpElement = this.dom.createElement('div');
-    const elements = this._renderGroupedAudits(auditRefs, groupDefinitions, {expandable: false});
+    const elements = this._renderGroupedAudits(auditRefs, groupDefinitions, {
+      expandable: false,
+      expandByDefault: false,
```

Comment:
this one is a bit confusing since won't it be expanded by default? how else would you expand it? maybe this value shouldn't matter when `expandable` is `false`